authui/src/authflowv2/components/settings-content.css

```diff
@@ -9,7 +9,9 @@
 
     @media (min-width: theme("screens.tablet")) {
       margin: 0 auto;
-      width: var(--settings-content__width);
+      width: calc(
+        var(--settings-content__width) + var(--widget-content__px, 0px)
```

Comment:
It is better to define `:root { --widget-content__px: 0px}` in `widget.css`, so in here, we do not need to provide a custom fallback value.